### ISSUE TITLE
Can't load any maps "Players failed to load content." (Both my own servers or others.)

### ISSUE THREAD

**Issue by SirZandro** [ElDewrito/ElDorito#474]
After downloading the complete 0.6 game from MEGA (that doesn't require any patching) I extracted it into its own folder. I quickly realized I was unable to enter any lobbies being my own (multiplayer and forge) or another players. I would receive the "Players failed to load content." error message and could not load any maps. I tried to download other maps after realization of this error to see if my game could see those instead (the 0.6 release map pack by Clutchism as well as MLG Maps and Variants by SimplyStats), they could not. I've tied downloading 0.6 from Mediafire as well, which also produced the same error. I was expecting to at least see the new maps appear in my in-game menus, but they do not. This leads me to believe that the reason I can't load any maps is because my install somehow can't see any of my files in either the maps or mods folders. I don't know how to reproduce the issue, however I loaded an exact copy of my install into my laptop and the issue ceased, the maps loaded correctly. This makes me believe that it is related so some personal pc setting or file complication on my gaming desktop.

I have tried the following to fix the issue: I made sure all files were NOT read-only, gave full control through the properties security tab, redownloading from more than one link, verifying all map files were there, updating graphics drivers, redownloading directx files, running executable as admin, restarting pc, port forwarding, whitelisting the maps and executables from my antivirus, running the game on a different disk,  messed with settings through nvidia control panel, altered various settings in dewrito_prefs.cfg.

**Comment by JoshuaGoode** [comment]
Same issue here. 

Looking into other possible conflicts.  To get started, I'm running Windows 10.

**Comment by JoshuaGoode** [comment]
Update:  My game now loads maps locally but  I must manually adjust dewrito_prefs.cfg; manually kill the game's process when I'm done; I cannot access the settings menu; and cannot access the server browser.  Prior to this, I was able to do all of the above but could not save settings/customizations made in game and could not load maps due to the same 'Players failed to load content' error.

I reinstalled using Method 2, ensured all files were not read-only, edited security settings for full control for my user, and **UNCHECKED** 'Run this program as an administrator' under eldorado.exe's settings.  Then, I **CHECKED** ''Run this program as an administrator' for custom_menu.exe.

This allows me to load maps locally via Host Multiplayer and Host Forge but I cannot navigate most of the other menus.

Before and after adjusting custom_menu.exe's settings, the custom_menu.log spits out
```
[0425/014724.169:WARNING:backend_impl.cc(1807)] Destroying invalid entry.
[0425/014724.224:ERROR:backend_impl.cc(751)] Unable to create file: -4
[0425/014724.224:ERROR:entry_impl.cc(964)] Failed to save user data
[0425/014724.227:WARNING:backend_impl.cc(1807)] Destroying invalid entry.
[0425/014724.247:ERROR:backend_impl.cc(751)] Unable to create file: -4
[0425/014724.247:ERROR:entry_impl.cc(964)] Failed to save user data
[0425/014724.248:ERROR:backend_impl.cc(751)] Unable to create file: -4
[0425/014724.248:ERROR:entry_impl.cc(964)] Failed to save user data
[0425/014724.262:WARNING:backend_impl.cc(1807)] Destroying invalid entry.
[0425/014724.281:ERROR:backend_impl.cc(751)] Unable to create file: -4
[0425/014724.281:ERROR:entry_impl.cc(964)] Failed to save user data
[0425/014724.305:ERROR:backend_impl.cc(751)] Unable to create file: -4
[0425/014724.305:ERROR:entry_impl.cc(964)] Failed to save user data
[0425/014724.635:WARNING:backend_impl.cc(1807)] Destroying invalid entry.
[0425/014724.715:WARNING:backend_impl.cc(1807)] Destroying invalid entry.
[0425/014724.755:ERROR:backend_impl.cc(751)] Unable to create file: -4
[0425/014724.755:ERROR:entry_impl.cc(964)] Failed to save user data
[0425/014724.757:ERROR:backend_impl.cc(751)] Unable to create file: -4
[0425/014724.757:ERROR:entry_impl.cc(964)] Failed to save user data
[0425/014724.786:WARNING:backend_impl.cc(1807)] Destroying invalid entry.
[0425/014724.789:WARNING:backend_impl.cc(1807)] Destroying invalid entry.
[0425/014724.789:WARNING:backend_impl.cc(1807)] Destroying invalid entry.
[0425/014724.962:ERROR:backend_impl.cc(751)] Unable to create file: -4
[0425/014724.962:ERROR:entry_impl.cc(964)] Failed to save user data
[0425/014724.964:ERROR:backend_impl.cc(751)] Unable to create file: -4
[0425/014724.964:ERROR:entry_impl.cc(964)] Failed to save user data
[0425/014724.985:ERROR:backend_impl.cc(751)] Unable to create file: -4
[0425/014724.985:ERROR:entry_impl.cc(964)] Failed to save user data
[0425/014725.227:ERROR:backend_impl.cc(751)] Unable to create file: -4
[0425/014725.227:ERROR:entry_impl.cc(964)] Failed to save user data
[0425/014725.231:WARNING:backend_impl.cc(1807)] Destroying invalid entry.
[0425/014725.318:WARNING:backend_impl.cc(1807)] Destroying invalid entry.
[0425/014725.396:ERROR:backend_impl.cc(751)] Unable to create file: -4
[0425/014725.396:ERROR:entry_impl.cc(964)] Failed to save user data
[0425/014725.405:WARNING:backend_impl.cc(1807)] Destroying invalid entry.
```

**Comment by SirZandro** [comment]
Windows 10. Was able to replicate your steps. Same issue, however my custom_menu.log is always empty.

**Comment by SirZandro** [comment]
Update: Sounds like someone in discord got both the maps to load and the menus working by setting both eldorado.exe and custom_menu.exe to run as administrator and then setting them both to run in Windows 7 compatibility mode. I was not so lucky, I replicated his solution and continue to have the same issues.

**Comment by blamay** [comment]
I am also having this issue. Tried running as administrator, tried windows 7 compatibility mode, tried installing the directx 9 june update & latest directx. It works just fine on another computer on the same network. Both computer are connected via ethernet.

**Comment by SirZandro** [comment]
Update: When both have 'run as administrator' unchecked and receive the "Players Failed to Load Content." error, I checked what processes were running and both eldorado.exe and custom_menu.exe appear. If I only check custom_menu.exe as run as administrator to get the local games running, I noticed that only 'ElDewrito | Version: 0.6.0.0 |Build Date: Apr 19 2018' is running and not custom_menu.exe. This would explain why the menus don't work for the process isn't even open. So this confirms that custom_menu.exe is causing the maps to fail to load, but have yet to find a fix.

**Comment by griffisYT** [comment]
Have the same issue, tried everything there is you can find on it, nothing has worked, at least one of the devs have to acknowledge this at some point

**Comment by hak33m16** [comment]
Can everybody here who is having the issue state their exact name and version of Windows 10? I'm not having any of these issues, and I'm running a Windows 10 EDU, version 10.0.16299 Build 16299. You can find this information by opening up your "System Information" program in Windows 10, and looking at the first and second entries on the right side.

**Comment by hak33m16** [comment]
The developer Twister on the discord helped me find out what those errors meant, and he pointed me to the following line of code in the CEF source: https://github.com/adobe/chromium/blob/master/net/disk_cache/backend_impl.cc#L1893 which means that it is actually an issue with chromium, which is what they use for their UI. Obviously not a fix, but could potentially lead to understanding why it's not working.

**Comment by griffisYT** [comment]
I don't know if this is by chance or an actual fix but i went to this website https://chromium.woolyss.com/ downloaded the most recent update and now my game is letting me load maps :OOO

**Comment by hak33m16** [comment]
@griffisYT I noticed you deleted your comment lol. Did it stop working again?

**Comment by griffisYT** [comment]
Yeah faulse alarm i forgot i put it in comp mode for windows 7 which allowed me to load only forge mode maps without anything else working and i got excited lol

**Comment by blamay** [comment]
If any devs know what the problem might be. It'd be really helpful to at least hear it. Perhaps a mod could fix this problem in the future. It really sucks that some of my friends just cannot play on their system.

**Comment by JoshuaGoode** [comment]
I no longer see Abdel-Hakeem Badran's comments on this issue.  For the record, I will post his most significant one below.

>...Twister on the discord helped me find out what those errors meant, and he pointed me to the following line of code in the CEF source: https://github.com/adobe/chromium/blob/master/net/disk_cache/backend_impl.cc#L1893 which means that it is actually an issue with chromium, which is what they use for their UI. Obviously not a fix, but could potentially lead to understanding why it's not working.

_**-Abdel-Hakeem Badran**_

**Comment by hak33m16** [comment]
I retract that statement as, according to Twister from the dev team, the "Players failed to load content." error is engine specific. As of now, he says they are unable to capture errors from the base game, and so the error "[is] as mystery". The errors in that log are due to CEF, yes, but they're not the cause of the issue, according to Twister himself.

**Comment by JoshuaGoode** [comment]
Thanks, @hak33m16!

That is good to know as I was already going down that path.  

Let's keep these comments here so we have record of what has been investigated and what paths we don't need to go down.

**Comment by SirZandro** [comment]
Recently updated to 0.6.1 (I know it was just a security patch but a man could hope) and still have the issue.
